views/partials/participant-evaluation-list.html.twig

```diff
@@ -3,43 +3,51 @@
     </div>
     <div class="panel-body">
-        {% for evaluation in evaluations %}
-            {% if type is not defined and loop.index == 6 %}
-                <div id="evaluation-overflow">
-            {% endif %}
-            <a href="{{ path('evaluation', { participantId: participant.id, evalId: evaluation.id }) }}" class="btn btn-block btn-lg {% if evalId is defined and evaluation.id == evalId %} btn-warning {% else %} btn-default {% endif %}">
-                {{ evaluation.created_ts|date('D n/j/Y', app.userTimezone) }}<br />
-                <small class="text-muted">
-                    {% if evaluation.eh_type == 'cancel' %}
-                        <span class="label label-danger">Cancelled</span> {{ evaluation.eh_created_ts|date('n/j/Y g:ia', app.userTimezone) }}
-                    {% elseif evaluation.rdr_id is empty and evaluation.parent_id is empty %}
-                        <span class="label label-default">Saved &amp; Unfinalized</span> {{ evaluation.updated_ts|date('n/j/Y g:ia', app.userTimezone) }}
-                    {% elseif evaluation.rdr_id and evaluation.parent_id %}
-                        <span class="label label-success">Edited &amp; Finalized</span> {{ evaluation.finalized_ts|date('n/j/Y g:ia', app.userTimezone) }}
-                    {% elseif evaluation.rdr_id %}
-                        <span class="label label-success">Finalized</span> {{ evaluation.finalized_ts|date('n/j/Y g:ia', app.userTimezone) }}
-                    {% elseif evaluation.parent_id and evaluation.rdr_id is empty %}
-                        <span class="label label-warning">Edited &amp; Unfinalized</span> {{ evaluation.updated_ts|date('n/j/Y g:ia', app.userTimezone) }}
-                    {% endif %}
-                </small>
-            </a>
-        {% endfor %}
-        {% if type is not defined and evaluations|length > 5 %}
-            </div> {# close .evaluation-overflow div above #}
-            <a href="#" class="btn btn-block btn-lg btn-default" id="evaluation-overflow-show">
-                <i class="fa fa-chevron-circle-down" aria-hidden="true"></i> Show all
-            </a>
-        {% endif %}
-        {% if type is not defined %}
-            <a href="{{ path('evaluation', { participantId: participant.id }) }}" class="btn btn-block btn-lg btn-default">
-                <i class="fa fa-plus-circle" aria-hidden="true"></i>
-                {% if evaluations|length > 0 %}
-                    New
+        {% if readOnly is defined and evaluations|length == 0 %}
+            <small class="text-warning">No Records Found</small>
+        {% else %}
+            {% for evaluation in evaluations %}
+                {% if type is not defined and loop.index == 6 %}
+                    <div id="evaluation-overflow">
+                {% endif %}
+                {% if readOnly is defined %}
+                    <a class="btn btn-block btn-lg btn-default readOnly">
```

Comment:
Change a to a div since this isn't a link. Also, I suggest changing the `readOnly` class name to something like `btn-readonly` to make it clear it is specifically for overriding the Bootstrap btn class. You'll then need to conditionally change the closing a/div below.